BHoM_UI/Components/Adapter/Push.cs

```diff
@@ -25,17 +25,16 @@
 using BH.oM.Adapter;
 using BH.oM.Reflection;
 using BH.oM.Reflection.Attributes;
-using BH.UI.Templates;
 using System;
 using System.Collections.Generic;
 using System.ComponentModel;
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
 
-namespace BH.UI.Components
+namespace BH.UI.Base.Components
```

Comment:
Wonder if the namespace change here means that the push method on this class needs a `PreviousVersion` attribute, as versioning from older version of this method points to a class that does not exist any longer?